libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/AudioFocusDelegateProvider.kt

```diff
@@ -1,23 +1,28 @@
 package com.mapbox.navigation.ui.voice.api
 
+import android.content.Context
 import android.media.AudioManager
 import android.os.Build
+import com.mapbox.navigation.ui.voice.options.VoiceInstructionsPlayerOptions
 
 internal object AudioFocusDelegateProvider {
```

Comment:
@abhishek1508 Maybe we should make this and both `PreOreoAudioFocusDelegate`, and `OreoAndLaterAudioFocusDelegate` classes public to allow developers to wrap our AsyncAudioFocusDelegate implementation?

Reply:
Ok, we can add this to `PreOreoAudioFocusDelegate` and `OreoAndLaterAudioFocusDelegate` as classes which interact with AudioManager